xdc-plugin/tests/counter.xdc

```diff
@@ -1,27 +1,27 @@
 #set_property LOC R2 [get_ports led]
 #OBUF_6
-set_property DRIVE 12 [get_ports {led[0]}]
+set_property DRIVE 12 [get_ports led[0]]
 #OBUF_7
-set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {led[1]}]
+set_property IN_TERM UNTUNED_SPLIT_40 [get_ports led[1]]
 set_property SLEW FAST [get_ports {led[1]}]
 set_property IOSTANDARD SSTL135 [get_ports {led[1]}]
 #OBUF_OUT
 set_property IN_TERM UNTUNED_SPLIT_50 [get_ports {out_a}]
-set_property SLEW FAST [get_ports {out_a}]
+set_property SLEW FAST [get_ports out_a]
 set_property IOSTANDARD LVCMOS33 [get_ports {out_a}]
 #bottom_inst.OBUF_10
 set_property SLEW SLOW [get_ports {out_b[0]}]
 set_property IOSTANDARD LVCMOS18 [get_ports {out_b[0]}]
 #bottom_inst.OBUF_11
 set_property DRIVE 4 [get_ports {out_b[1]}]
 set_property SLEW FAST [get_ports {out_b[1]}]
-set_property IOSTANDARD LVCMOS25 [get_ports {out_b[1]}]
+set_property IOSTANDARD LVCMOS25 [get_ports out_b[1]]
```

Comment:
Why does this line change but not line 21?

Reply:
This is because `get_ports out_b[1]` should be equivalent to `get_ports {out_b[1]}` hence the test contains both versions in random places.